lib/src/controls/yaru_popup_menu_button.dart

```diff
@@ -9,86 +9,87 @@ class YaruPopupMenuButton<T> extends StatelessWidget {
     super.key,
     required this.initialValue,
     required this.child,
-    required this.items,
+    required this.itemBuilder,
     this.onSelected,
     this.onCanceled,
     this.tooltip,
     this.position = PopupMenuPosition.under,
+    this.childPadding = const EdgeInsets.symmetric(horizontal: 5),
```

Comment:
by definition, ["padding"](https://www.w3schools.com/css/css_boxmodel.asp) is inside the border - why is this called "child" padding? also, this API is a bit unpredictable because the padding is in fact inserted twice.